fragment_buffer_test.go

```diff
@@ -47,12 +47,12 @@ func TestFragmentBuffer(t *testing.T) {
 			}
 		}
 
-		out := fragmentBuffer.pop()
+		out, _ := fragmentBuffer.pop()
```

Comment:
Can you assert the `uint16` as well, just to make sure that doesn't regress/break.

Reply:
Added an assert of returned epoch and an another test data having non-zero epoch.